fdbcli/fdbcli.actor.cpp

```diff
@@ -581,6 +581,11 @@ void initHelp() {
 		"view and control throttled tags",
 		"Use `on' and `off' to manually throttle or unthrottle tags. Use `enable auto' or `disable auto' to enable or disable automatic tag throttling. Use `list' to print the list of throttled tags.\n"
 	);
+	helpMap["cache_range"] = CommandHelp(
```

Comment:
How long does it take to set one cache range?  (Is it the latency of write transaction?)
If multiple shards are hot, we don't want to take too long to set the cache range.